server/controllers/finance/reports/collection_capacity/index.js

```diff
@@ -0,0 +1,102 @@
+/**
+ * @overview ./finance/reports/collection_capacity/
+ */
+
+const _ = require('lodash');
+const moment = require('moment');
+
+const ReportManager = require('../../../../lib/ReportManager');
+const db = require('../../../../lib/db');
+
+module.exports.report = report;
+
+// path to the template to render
+const TEMPLATE = './server/controllers/finance/reports/collection_capacity/report.handlebars';
+
+const DEFAULT_OPTIONS = {
+  orientation : 'portrait',
+  footerRight : '[page] / [toPage]',
+  footerFontSize : '7',
+};
+
+/**
+ * @method report
+ *
+ * @description
+ * The HTTP interface which actually creates the report.
+ */
+async function report(req, res, next) {
+  try {
+    const qs = _.extend(req.query, DEFAULT_OPTIONS);
+    const {
+      dateFrom, dateTo, accountId, cashboxId,
+    } = req.query;
+    const metadata = _.clone(req.session);
+
+    const rpt = new ReportManager(TEMPLATE, metadata, qs);
+
+    const CASH_PAYMENT_TRANSACTION_TYPE = 2;
+    const CAUTION_TRANSACTION_TYPE = 19;
+
+    const patients = `
+      SELECT DATE(p.registration_date) AS registration_date, COUNT(*) registrations
+      FROM patient p 
+      WHERE (DATE(p.registration_date) BETWEEN ? AND ?)
+        AND project_id = ?
```

Comment:
I think you are filtering on project because you want to capture the difference between PAX and Tshikaji.  I don't think it makes sense to do the filtering on project though.   Nothing in BHIMA says that invoices made in one project _must_ be paid back in that same project.  With this filtering we will probably lose information.  Let's just look at the entire institution instead.